src/app/App.svelte

```diff
@@ -10,8 +10,10 @@
   import ChromeService from '../services/ChromeService.js'
   import CompPlexService from '../services/CompPlexService'
   import { WIDGET_ID, EVENTS } from '../constants'
+  import ErrorCallout from './components/ErrorCallout.svelte'
 
   let loading = false
+  let error = true
```

Comment:
Depending on how the reviews go, I might merge this, but will remove before creating a release.